package/MDAnalysis/coordinates/chain.py

```diff
@@ -44,7 +44,6 @@
 import warnings
 
 import os.path
-import itertools
```

Comment:
Hah – I didn't realize that we weren't using it here anymore... the original code had a `itertools.chain([... readers ...])` at its heart.